source/ClusterAPI/Setup.rst

```diff
@@ -33,7 +23,7 @@ to quickly gain access and perform recovery or upgrades as required.
 Account Security
 ----------------
 
-A `clouds.yaml` file with the user's password is also required. This file should be removed or restricted
+A `clouds.yaml` file with the application credentials is also required. This file should be removed or restricted
```

Comment:
Should we also mention for application credentials when they are created they should **never** be unrestricted application credentials?